async-voter.js

```diff
@@ -51,9 +51,10 @@ module.exports = (app, repository) => {
     const channel_id = req.body.channel_id
 
     // TODO: Close previous session. One session per channel is allowed.
-    repository.del(channel_id, (err, reply) => {
+    repository.del(channel_id + "-" + text + "-initiation", (err, reply) => {
```

Comment:
I wonder if we need to delete the previous session with this new setup?  I guess it ensures that any existing vote with that precise text is deleted - but in the case of a re-vote, we might want to alert the users to the fact that this is re-voting on something that was previously voted on.  Perhaps that's another feature ...?

At the very least we should update the comment above, which doesn't look like a TODO and is now no longer accurate, since with this new setup we can support more than one session per channel can't we ...